platform-ui-skin/src/main/webapp/skin/less/social/skin/social.less

```
@@ -514,4 +514,16 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 }
 .uiTableProfileUserSearchInput {
     width: 100%;
+}
+.uiNotFoundActivity {
+	text-align: center;
+	margin: 200px auto auto;
+	.notfoundActivity {
+		background: url('@{images-path}/social/skin/Activity/notfoundActivity.png') no-repeat top center;
+		padding-top: 90px;
+		padding-left: 120px;
+	}
+	h4 {
+		margin:20px;color:#a8b3c5
```

```suggestion
		margin:20px;
		color:#a8b3c5
```
See if we have a color variable